src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -275,105 +285,63 @@ For an example of how to build an internal integration, see [our Round Robin Iss
 
 Creating an internal integration will automatically install it on your organization.
 
-#### Auth Tokens
+### Auth Tokens
 
 Auth Tokens are what the Integration Platform calls tokens. Auth Tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
```

Comment:
Same comment as the same spot in the public integrations content: I'm confused by this bit. Lemme know if this edit doesn't work and we might need to talk it through.

```suggestion
Auth Tokens are referred to as _tokens_ in the Integration Platform. Auth tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
```